docs/general/ddl/data-types.rst

```diff
@@ -82,28 +82,116 @@ A basic boolean type. Accepting ``true`` and ``false`` as values. Example::
     cr> drop table my_bool_table;
     DROP OK, 1 row affected (... sec)
 
+.. _character-data-types:
+
+Character Types
+===============
+
+The general-purpose character data types available in CrateDB.
+
+Columns of below listed character data types can also be analyzed. By default
+the :ref:`plain <plain-analyzer>` analyzer is used. See
+:ref:`sql_ddl_index_fulltext`.
+
+.. _data-type-varchar:
+
+``character varying(n)``, ``varchar(n)``
+----------------------------------------
+
+The ``character varying(n)`` or ``varchar(n)`` character data types represent
+variable length strings. All unicode characters are allowed.
+
+The optional length specification ``n`` is a positive `integer <numeric
+types_>`_ that defines the maximum length, in characters, of the values
+that have to be stored or cast. The minimum length is ``1``. The maximum
+length is defined by the upper `integer <numeric types_>`_ range.
+
+An attempt to store a string literal that exceeds the specified length
+of the character data type results in an error.
+
+::
+
+    cr> CREATE TABLE users (id varchar, name varchar(6));
+    CREATE OK, 1 row affected (... sec)
+
+::
+
+    cr> INSERT INTO users (id, name) VALUES ('1361', 'john doe')
+    SQLActionException[SQLParseException: 'john doe' is too long for the text type of length: 6]
+
+Unless the excess characters are all spaces, in this case the string literal
+will be truncated to the specified length.
```

Comment:
If the excess characters are all spaces, the string literal will be truncated 
to the specified length.